internal/commands/build/build.go

```diff
@@ -39,7 +42,7 @@ type buildOptions struct {
 	noCache        bool
 	progress       string
 	pull           bool
-	tag            string
+	tags           cliOpts.ListOpts
```

Comment:
for homogeneity `args` should be a `cli.opts.ListOpt` as well

Reply:
Done